ecs_deploy/ecs.py

```diff
@@ -7,9 +7,10 @@
 
 class EcsClient(object):
     def __init__(self, access_key_id=None, secret_access_key=None,
-                 region=None, profile=None):
```

Comment:
Is there a reason, why the new argument is not added at the end? That would be more backwards compatible, as even in the tests the arguments are not passed with keywords